packages/oc-azure-storage-adapter/index.js

```diff
@@ -199,7 +199,8 @@ module.exports = function(conf) {
           const relativeFile = file.substr(dirInput.length),
             url = (dirOutput + relativeFile).replace(/\\/g, '/');
 
-          putFile(file, url, relativeFile === '/server.js', cb);
+          const serverJsNames = ['/server.js', '\\server.js'];
```

Comment:
too DRY to move `serverJsNames` in a common/shared file?

Reply:
hmm... that would need to be two separate PR's I believe since it would have to go into utils which is it's own npm package, unless we can somehow release it in a controlled manner.

@matteofigus what do you think?

Reply:
it should be part of this monorepo or am I missing something? in any case, it still looks good to me :)